packages/bento-frontend/src/bento/fileCentricCartWorkflowData.js

```diff
@@ -40,33 +53,54 @@ export const manifestData = {
   header: ['Study Code', 'Case ID', 'File Name', 'File ID', 'Md5sum', 'User Comments'],
 };
 
+// --------------- GraphQL query - Retrieve selected cases info --------------
```

Comment:
Can we move query at the very bottom of the file, same like all the other custodian configurations follow?